www/scripts/codecheckerviewer/CheckerStatistics.js

```diff
@@ -129,6 +140,81 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
       this._populateStatistics();
     },
 
+    onRowClick : function (evt) {
+      var that = this;
+      var item = this.getItem(evt.rowIndex);
+
+      var checkerNameFilter = that.bugFilterView._checkerNameFilter;
+      var reviewStatusFilter = that.bugFilterView._reviewStatusFilter;
+      var detectionStatusFilter = that.bugFilterView._detectionStatusFilter;
+      var severityFilter = that.bugFilterView._severityFilter;
+
+      var state = that.bugFilterView.clearAll();
+      state[checkerNameFilter.class] = item.checker;
+
+      switch (evt.cell.field) {
+        case 'checker':
+          state[reviewStatusFilter.class] =
+          Object.keys(CC_OBJECTS.ReviewStatus).map(function (key) {
+            return reviewStatusFilter.stateConverter(
+              CC_OBJECTS.ReviewStatus[key]);
+          });
+
+          state[detectionStatusFilter.class] =
+          Object.keys(CC_OBJECTS.DetectionStatus).map(function (key) {
+            return detectionStatusFilter.stateConverter(
+              CC_OBJECTS.DetectionStatus[key]);
+          });
+          break;
+        case 'severity':
+          state[severityFilter.class] =
+            severityFilter.stateConverter(item.severity[0]);
+          break;
+        case 'reports':
+          // Show all reports
+          break;
+        case 'resolved':
+          if (!item.resolved[0]) return;
+
+          state[detectionStatusFilter.class] =
+          detectionStatusFilter.stateConverter(
+            CC_OBJECTS.DetectionStatus.RESOLVED);
+          break;
+        case 'unreviewed':
+          if (!item.unreviewed[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
+            CC_OBJECTS.ReviewStatus.UNREVIEWED);
+          break;
+        case 'confirmed':
+          if (!item.confirmed[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
+            CC_OBJECTS.ReviewStatus.CONFIRMED);
+          break;
+        case 'falsePositive':
+          if (!item.falsePositive[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
+            CC_OBJECTS.ReviewStatus.FALSE_POSITIVE);
+          break;
+        case 'intentional':
+          if (!item.intentional[0]) return;
+
+          state[reviewStatusFilter.class] =reviewStatusFilter.stateConverter(
```

Comment:
Is this formatting conventional for javascript? (No space after assignment)

Reply:
No, this is definitely not. Thanks.